### DISCUSSION TITLE
Google Mail integration - Sensitive/restrictive scope

### DISCUSSION THREAD

**Discussion by sreekanthedayar** [jstedfast/MimeKit#1063] [Q&A]
Hi, We are planning to implement a team inbox feature with the ability to add gmail/google workspace email to manage it with team. After going through the gmail API/SMTP/imap integration instruction, we understand that, for integrating anything related to Gmail, we need to undergo a 3rd party audit which is costly. Now the option left is to use API like nylas or unipile. Can somebody confirm this or is there any workaround to use mimekit to integrate Gmail at the moment?

**Comment by jstedfast** [comment]
> we understand that, for integrating anything related to Gmail, we need to undergo a 3rd party audit which is costly

I had no idea about that :(

> Now the option left is to use API like nylas or unipile. Can somebody confirm this or is there any workaround to use mimekit to integrate Gmail at the moment?

I am not familiar with nylas or unipile.

What, exactly, are you trying to do? I'm not really sure what a "team inbox feature" is or what you mean by "adding a gmail/google workspace email to manage it with a team."

Maybe if I better understood what you were trying to do, I could offer some suggestions.
